zomes/blog/code/src/blog.rs

```diff
@@ -84,52 +76,54 @@ pub fn handle_create_post(content: String, in_reply_to: HashString) -> serde_jso
                     let _ = hdk::link_entries(&in_reply_to, &post_hash, "comments");
                 }
             }
-            json!({"hash": post_hash})
+            json!({"address": post_hash}).into()
```

Comment:
this seems weird to me, manually using `json!` instead of making a struct that derives `DefaultJson` and using that

Reply:
Sure. I managed to get rid of all uses of the json! macros by adding two extra structs